dask/dataframe/io/tests/test_parquet.py

```diff
@@ -2789,7 +2918,7 @@ def test_read_pandas_fastparquet_partitioned(tmpdir, engine):
     pdf.to_parquet(path, partition_cols=["group"], engine="fastparquet")
     ddf_read = dd.read_parquet(path, engine=engine)
 
-    assert len(ddf_read["group"].compute()) == 6
+    assert len(ddf_read["group"].compute(scheduler="synchronous")) == 6
```

Comment:
debugging, I assume? is this here to stay?

Reply:
My mistake.